rmw_fastrtps_shared_cpp/src/participant.cpp

```diff
@@ -13,7 +13,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <fstream>
 #include <memory>
+#include <sstream>
```

Comment:
I don't think we need to add these headers

```suggestion
#include <memory>
```

Reply:
Changes applied on c289b49